pom.xml

```diff
@@ -53,9 +53,10 @@
         <maven.surefire.version>2.22.0</maven.surefire.version>
 
         <google.guava.version>20.0</google.guava.version>
+        <google.gson.version>2.8.6</google.gson.version>
```

Comment:
Why are you using an additional json framework? Jackson is already used internally by Spring.